governance/policies/archiving.md

```diff
@@ -19,7 +17,7 @@
 
 The current compiler cutoff threshold is `4.08`.
 
-This threshold is subject to change by the opam repo maintainers based on the oldest compiler available on maintained distributions.
+This threshold is subject to change by the opam repo maintainers based on the oldest compiler available on the maintained[^1] distributions tested in the opam-repository CI.
```

Comment:
In particular I don't really like this sentence :P

Reply:
Would you like it better if it were refactored into a couple shorter sentences? E.g.,

```suggestion
This threshold is subject to change by the opam repo maintainers.
The threshold is based on the oldest ocaml compiler version available 
in the maintained[^1] distributions. It determines the minimum
compiler version used in tests for the opam-repository CI[^2].

[^2]: https://github.com/ocurrent/opam-repo-ci/blob/master/doc/platforms.md#ocaml-versions
```